databases/week3/queries.sql

```diff
@@ -0,0 +1,85 @@
+--Meal
+--Get all meals
+SELECT * FROM meal;
+--Add a new meal
+INSERT INTO `meal` (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`) VALUES (1,'Palak Paneer', 'Paneer cubes in palak grvay', 'spice kitchen,Lyngby', '2022-05-10 10:00:00', 10, 325.25, '2022-05-10');
```

Comment:
Usually it's a bad idea to insert the primary key (id in this case) manually, the system does it automatically when you create the column with "AUTO_INCREMENT", and that way we never get errors because we manually inserted a value that already exists.